src/components/organisms/JoinVenueModal/JoinVenueModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC, Fragment, useCallback } from "react";
```

Comment:
[nit]

```suggestion
import React, { useCallback } from "react";
```

Reply:
@sparkletown/engineering 
I'd like to say that I prefer to use named fragments because of the ability to use `key`. Sometimes you need it and we should keep things consistent. I'd also like to say that I'd prefer if we directly import Fragment and FC instead of using the dot syntax `React.FC`. They are named exports and we should take advantage of that, we don't import hooks using `React.useState` why import FC with it?. Of course, that's just my personal preference. I believe it's up to @0xdevalias to decide if that makes sense or not.

Reply:
I have no strong feelings on the shorthand fragment syntax vs the longhand. In the past I actively chose to use the longhand form, as it is more explicit/'self-documenting' to other users as to what it is (eg. it's hard to do a google search for `<>` and find meaningful results, but searching for 'react fragment' will give you much better insight)

I personally don't feel that 'absolute consistency' is required, if we decided that the shorthand is better for most cases, and then we only used the longhand form where it is required (which is pretty much only when we need to add `key` to it I believe) In many cases where we would want to add `key` to it though, I would question if Fragment was the right choice, as off the top of my head, I can't think of many use cases where 'smooshing' an array of things together without containing divs/similar to 'group' them feels 'right' (i'm sure there are cases I'm not immediately thinking of though)

My preference towards using `React.Fragment` and `React.FC` is that I feel like it provides more 'context'/etc to describe what it is. I'm a little bit either way on `React.Fragment` (my preference in the past was to use the `React.` prefix explicitly, I'm not sure if I feel particularly strongly about that anymore); but with `React.FC`, for the same reason that I prefer variable names to be 'descriptive', I feel like `FC` is too 'obscure' on it's own, and spelling out the full `FunctionalComponent` version of it feels overly verbose; so for me, `React.FC` seems to fit that middleground of terse, yet descriptive.

Reply:
(captured in https://github.com/sparkletown/sparkle/pull/1062#issuecomment-776410919)